docs/source/etherlime-cli.rst

```diff
@@ -97,8 +97,11 @@ Compiling
 Parameters:
 
 * ``dir`` - [Optional] By specifying ``dir`` you can set the root directory where to read the contracts and place the build folder. By default ``dir`` is set to the current working directory ``./``
+<<<<<<< HEAD
```

Comment:
Once you are done you should remove the message about library linking support (as we would obviously supporting it). Also do not forget to update the docs to indicate we are ready with the compilation (both here in the docs and in the README.md file)